README.md

```diff
@@ -88,3 +88,33 @@ After you've started the dev server, you can open the interactive browser tests
 ```sh
 npm run cypress:open
 ```
+
+### Chore: Dependency Updates
+
+You can see which dependencies have new releases by first making sure your local dependencies are up-to-date by executing `npm ci` and then running `npm outdated`.
+The following dependencies should be ignored:
+
+- Wikit (i.e. `@wmde/wikit-tokens` and `@wmde/wikit-vue-components`):
+  we’re using a newer pre-release version and don’t want to downgrade to the latest full release.
+- Vue and Vuex:
+  in production, we use the versions shipped by MediaWiki core,
+  so we should use the same versions for testing.
+
+All other dependencies should generally be updated to the latest version.
+If you discover that a dependency cannot be updated for some reason, please add it to the above list.
```

Comment:
I think it would be better to emphasize that this is a judgement call, not a limitation:

```suggestion
If you discover that a dependency should not be updated for some reason, please add it to the above list.
```